test/functional/test-3p-environment.js

```diff
@@ -35,6 +35,10 @@ describe('3p environment', () => {
     });
   });
 
+  afterEach(() => {
+    testWin = null;
```

Comment:
Actually, we're [deleting the tests](https://github.com/mochajs/mocha/pull/2037/files#diff-b3b53682a18f203ac8d29b0e277cad26R728), too. We already have everything https://github.com/mochajs/mocha/issues/1991#issuecomment-168379139 suggests.